package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Thunk middleware for Redux.",
   "main": "lib/index.js",
   "scripts": {
-    "prepublish": "rm -rf lib && babel src --out-dir lib"
+    "prepublish": "rm -rf lib && babel src --out-dir lib",
```

Comment:
Let's use `rimraf` like [here](https://github.com/gaearon/redux/blob/a467c4174ecd99e335b5160c1ce163141d438844/package.json#L7-L21) so it builds on Windows too.